__tests__/postArticleComment.test.js

```diff
@@ -0,0 +1,56 @@
+const request = require("supertest");
+const app = require("../index");
+const db = require("../db/connection");
+const seed = require("../db/seeds/seed");
+const data = require("../db/data/test-data");
+const jestSorted = require("jest-sorted");
+
+afterAll(() => {
+    db.end();
+  });
+  
+  beforeEach(() => {
+    return seed(data);
+  });
+
+  describe('POST /api/articles/:article_id/comments', () => {
+    it('Checks if the user has been created', async () => {
```

Comment:
I think this was because i misunderstood and i actually made a query to the DB to match the user i didn't realise SQL would do that for us until i got to that part so I've removed the other function but i must have missed this description to remove also, same with the is(NaN) functionality i'll remove those parts